config.toml.dist

```diff
@@ -51,3 +51,10 @@ secret =
 discovery_url = "https://accounts.google.com/.well-known/openid-configuration"
 # Google OpenID Connect id_token issuer domain - Default: "accounts.google.com"
 issuer_domain = "accounts.google.com"
+
+
+[limits]
+# Limit the number of session of any kind.
+auth = ["per-second:3:1:1", "per-minute:20:60:5"]
+# Limit the number of email loop authentications.
+auth_email = ["per-second:1:1:1", "per-minute:10:60:5"]
```

Comment:
I wonder if the benefits of this degree of granularity are worth the complexity / cognitive overhead?

E.g., Could we automatically generate a reasonable granularity value? Do we need names for each limit, even though they might look like magic strings and confuse people? Should we deliberately turn them into magic strings? Do we need a vector, or could we get by with a single limit for each case?

Could we get by with a single stupid value like this, and would it be better?

```
[smtp]
per_user = "10/min"
```

I genuinely don't know what the best answer is to these would be. I do know that parsing the limits section hurts my head a little more than I'd like, but not enough to block merging of a patch that *does* work, as long as you're generally happy with this design.